File: solutions/1505-create-target-array-in-the-given-order/solution.py
class Solution(object):
    def createTargetArray(self, nums, index):
        """
        :type nums: List[int]
        :type index: List[int]
        :rtype: List[int]
        """
        ls=[]
        j=0
        for i in index:
            ls.insert(i,nums[j])
            j+=1
        return ls

File: solutions/1505-create-target-array-in-the-given-order/solution.java
class Solution {
    public int[] createTargetArray(int[] nums, int[] index) {
        List<Integer> ls=new ArrayList<>();
        for(int i=0;i<index.length;i++)
            ls.add(index[i],nums[i]);
        int a=0;
        for(int n:ls)
            nums[a++]=n;
        return nums;
    }
}
